### ISSUE TITLE
Move 'Buildable' type class from 'text-show' library to 'text' library?

### ISSUE THREAD

**Issue by ChShersh** [haskell/text#214]
Duplicating existing issue from other package:

* https://github.com/bos/text-format/issues/14

Relevant recent Reddit discussion:

* https://www.reddit.com/r/haskell/comments/7q8kkd/uniform_buildable_type_class_for_formatting/

**Comment by gromakovsky** [comment]
`text-show` or `text-format`? I couldn't find `Buildable` in `text-show` :/

**Comment by ChShersh** [comment]
@gromakovsky Yes, you're right. Dunno what I was thinking about when I was writing this title...
